docs/UserGuide.md

```diff
@@ -778,7 +869,8 @@ Exit the application.
 
 ## Warning
 
-:warning: If your changes to the data file makes its format invalid, Le Tracker will discard all data and start with an empty data file at the next run.
+:warning: If your changes to the tracker data file makes its format invalid, Le Tracker will discard all data and 
+start with an empty data file at the next run.
```

Comment:
Understand that you're trying to line wrap your text. However, best to check if they will render out correctly after line wrapping due to markdown using line wrap as a form of formatting (it isn't like java where new lines are ignored and lines only end with `;`).

Manual line wrapping in markdown can become problematic especially when you're doing it to point form lines (lines that start with `-`). If you're not too sure how to handle all these properly, using no line wrap is probably fine I think. Don't think the 120 line width limit applies to markdown files.